ui/src/containers/Layout.js

```diff
@@ -143,14 +128,44 @@ const Layout = props => {
     },
   ];
 
+  const filterLanguage = languages.find(lang => lang.name !== language);
+
+  const rightActions = [
+    {
+      type: 'dropdown',
+      text: language,
+      icon: <i className="fas fa-globe" />,
+      items: filterLanguage === null ? null : [filterLanguage],
+    },
+    {
+      type: 'dropdown',
+      icon: <i className="fas fa-th" />,
+      items: applications === null ? null : [applications],
```

Comment:
`items: applications` is enough because applications is an array. if `applications` is null, you should not add it to `rightActions`